Lesson 11/src/index.js

```diff
@@ -0,0 +1,40 @@
+// string length
+
+const s = 'agdjgfkshv'
+console.log(s.length)
+
+// string reverce
+
+function reverseStr(a) {
+  return a.split('').reverse().join('')
+}
+
+console.log(reverseStr(s))
+
+// substr
+
+function findSubStr(str, substr) {
+  if (str.indexOf(substr) !== -1) {
+    console.log(`в строке ${str} есть подстрока ${substr}`)
+  } else console.log(`в строке ${str} нет подстроки ${substr}`)
```

Comment:
```suggestion
  } else {
    console.log(`в строке ${str} нет подстроки ${substr}`)
  }
```

записывай условия с новой строки в фигурных
(странно, что eslint тебе не высказал 😅)